src/views/Home.vue

```diff
@@ -157,13 +157,10 @@ export default defineComponent({
   height: 200px;
 }
 
- .internal {
-   margin-top: 24px;
```

Comment:
We want to have some margin here so that the internal section is a little distanced from the external. Spacer base maybe

Reply:
Done sir